packages/console/src/components/ToggleThemeButton/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { AppearanceMode } from '@logto/schemas';
+import classNames from 'classnames';
+
+import Moon from '@/assets/images/moon.svg';
+import Sun from '@/assets/images/sun.svg';
+
+import type { Props as ButtonProps } from '../Button';
+import Button from '../Button';
+import * as styles from './index.module.scss';
+
+type Mode = Omit<AppearanceMode, AppearanceMode.SyncWithSystem>;
+
+type Props = {
+  value: Mode;
+  onToggle: (value: Mode) => void;
+  className?: string;
+  iconClassName?: string;
+  size?: ButtonProps['size'];
+};
+
+const ToggleThemeButton = ({
+  value,
+  onToggle,
+  className,
+  iconClassName,
+  size = 'medium',
+}: Props) => {
+  const ThemeIcon = value === AppearanceMode.LightMode ? Sun : Moon;
+
+  return (
+    <div className={classNames(styles.container, styles[size])}>
+      <Button
```

Comment:
If I remember correctly, we do have an IconButton component?

Reply:
It's different from the `IconButton`:
<img width="124" alt="image" src="https://user-images.githubusercontent.com/10806653/222336870-fb16f140-43c5-4aa0-a856-f74abf5bc3c5.png">

It's a 'button with icon' here.